src/test/java/no/unit/nva/model/contexttypes/ChapterTest.java

```diff
@@ -0,0 +1,75 @@
+package no.unit.nva.model.contexttypes;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.net.URI;
+
+import static java.util.Objects.nonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ChapterTest {
+
+    public static final String TEST_URI = "https://example.org/123123";
+    public static final String LINKED_CONTEXT_TEMPLATE = "{\n  \"type\" : \"Chapter\",\n"
+            + "  \"linkedContext\" : \"%s\"\n}";
+    public static final String LINKED_CONTEXT_NULL_TEMPLATE = "{\n  \"type\" : \"Chapter\",\n"
+            + "  \"linkedContext\" : %s\n}";
+    private final ObjectMapper objectMapper = JsonUtils.objectMapper;
+
+    @DisplayName("ObjectMapper deserializes Chapter")
+    @Test
+    void objectMapperDeserializesChapterWhenTheInputIsValid() throws JsonProcessingException {
+        URI expectedLinkedContext = URI.create(TEST_URI);
+        String json = generateChapterJson(expectedLinkedContext.toString());
+        Chapter chapter = objectMapper.readValue(json, Chapter.class);
+        assertEquals(expectedLinkedContext, chapter.getLinkedContext());
+    }
+
+    @DisplayName("Chapter: ObjectMapper throws JsonMappingException when linkingContext URI is null")
+    @ParameterizedTest
+    @NullSource
+    void objectMapperThrowsJsonMappingExceptionWhenInputIsNull(String input) {
+        JsonMappingException exception = assertThrows(JsonMappingException.class,
+            () -> objectMapper.readValue(generateChapterJson(input), Chapter.class));
+
+        assertTrue(exception.getMessage(). contains(String.format(Chapter.ERROR_TEMPLATE, input)));
```

Comment:
1. put the expectedString as a separate variable. Let the compiler do the optimization for us.
2.  When this test fails the error message will be expected `true` but found `false`.  Suggestion: 
```
build.gradle:
   testImplementation group: 'org.hamcrest', name: 'hamcrest', version: '2.2'

ChapterTest:
String expectedErrorMessage= String.format(Chapter.ERROR_TEMPLATE, input);
assertThat(exception.getMessage(), containsString(expectedErrorMessage));
```